File: docs/test_questions.py
from questions import log_user_question

# Test data
user_id = "test_user_001"
video_id = "test_vid_001"
question = "What is LangChain?"
answer = "LangChain is a framework for building LLM-powered apps."
language = "en"

# Run test
result = log_user_question(user_id, video_id, question, answer, language)
print("Question logged:", result)

File: docs/test_questions.js
import { logUserQuestion } from './questions.js'; // Adjust path if needed

// Test data
const userId = "test_user_001";
const videoId = "test_vid_001";
const question = "What is LangChain?";
const answer = "LangChain is a framework for building LLM-powered apps.";
const language = "en";

// Run test
logUserQuestion(userId, videoId, question, answer, language)
  .then(result => {
    console.log("Question logged:", result);
  })
  .catch(error => {
    console.error("Error logging question:", error);
  });
